### ISSUE TITLE
False Positive for MultilineScript with =~

### ISSUE THREAD

**Issue by dmarcoux** [brigade/haml-lint#275]
Hey,

I have this code which triggers the MultilineScript linter:

```
  - if line =~ /\w+(?:\.\w+)+: W: /
    = content_tag(:span, line.strip, style: 'color: olive;')
```

The output from haml-lint is the following: 
    
    [W] MultilineScript: Script with trailing operator `/` should be merged with the script on the following line

I believe this shouldn't trigger the MultilineScript linter. It's not a big deal because the code could be written like this to get rid of the false positive.
```
  - if line.match?(/\w+(?:\.\w+)+: W: /)
    = content_tag(:span, line.strip, style: 'color: olive;')
```

Cheers

**Comment by Ana06** [comment]
This seems a bug with the linter... :thinking: What about reporting it upstream? It could fail again... :unamused:

**Comment by michaelherold** [comment]
I cannot get this to reproduce. I tried to do a `git bisect` to find where it failed, but haven't found a commit where this fails either.

Could you please write up a test that fails to help? Here's what I have and it's not raising the lint:

```ruby
  # within spec/haml-lint/linter/multiline_script_spec.rb
  context 'with a /-delimited regex on a line' do
    let(:haml) do
      <<-HAML
        - if line =~ /\w+(?:\.\w+)+: W: /
          = content_tag(:span, line.strip, style: 'color: olive;')
      HAML
    end

    it { should_not report_lint }
  end
```

Sorry for the trouble!

**Comment by dmarcoux** [comment]
I would give it a try, but how do I setup the project for development? I tend to work with Docker-based environments only. I could send a PR for the Docker setup. Let me know what you think.

**Comment by michaelherold** [comment]
Here's a minimal Dockerfile that could get you going:

```dockerfile
FROM ruby:2.5.1-alpine

WORKDIR /usr/src/app

COPY . ./

RUN apk add --update build-base && \
    bundle install --jobs 4 && \
    rm -rf /usr/share/ri && \
    apk del build-base
```

You can build the image like so (from the Git repo):

    docker build --rm -t haml-lint .

And then run the test suite like this:

    docker run --rm haml-lint sh -c "rspec"

**Comment by dmarcoux** [comment]
I gave it a try and I also cannot reproduce it... I will close this issue and reopen it if it happens again.

**Comment by Ana06** [comment]
@dmarcoux have you tried with the version of obs where supposedly that happened? :thinking:

**Comment by dmarcoux** [comment]
@Ana06 I tried with the same `haml-lint` version.
